simdjson/__init__.pyi

```diff
@@ -0,0 +1,108 @@
+import json
+from typing import (
+    AbstractSet,
+    Any,
+    Dict,
+    Final,
+    Iterator,
+    List,
+    Literal,
+    MutableMapping,
+    NewType,
+    Optional,
+    Tuple,
+    Union,
+    ValuesView,
+)
+
+
+class Object(MutableMapping[Any, Any]):
+    def __delitem__(self, key: Any) -> None:
+        ...
+
+    def __getitem__(self, key: Any) -> Any:
+        ...
+
+    def __setitem__(self, key: Any, value: Any) -> None:
+        ...
+
+    def __iter__(self) -> Iterator[Any]:
+        ...
+
+    def __len__(self) -> int:
+        ...
+
+    def as_dict(self) -> Dict[Any, Any]:
+        ...
+
+    def at_pointer(self, key: str) -> Any:
+        ...
+
+    def get(self, key: str, default: Any = None) -> Any:
+        ...
+
+    def keys(self) -> AbstractSet[Any]:
+        ...
+
+    def values(self) -> ValuesView[Any]:
+        ...
+
+    def items(self) -> AbstractSet[Tuple[Any, Any]]:
+        ...
+
+    @property
+    def mini(self) -> 'Object':
+        ...
+
+
+Buffer = NewType('Buffer', bytes)
+
+
+class Array(List[Any]):
```

Comment:
Like Object, it would rather be something like `List[Union[Primitives, List, Array]]`

Reply:
Hm, and probably Sequence not List, since we don't support mutation.

Reply:
Got it. I didn't know (or missed that) mutation is off the grid. Will adjust that.